src/mudsys/mud56.stink

```diff
@@ -0,0 +1,36 @@
+MPUREL
```

Comment:
The default FN2 for STINK scripts passed on the command line is LOADER.  But since STINK "2" doesn't take a JCL, it doesn't matter much now.